### PR TITLE
Yirou Fang HW1

### PR DESCRIPTION
**Pull request by yirouf**
I updated and added my self-introduction into my README file. It is not very difficult, but the entire process of moving back and forth between git, Rstudio, and local is a little bit complex at first.
After writing out the introduction, I went and looked at the RMarkdown cheatsheet. I think the cheat sheet is very thorough. I incorporated italic, bold, link, list (with sublist), blockquote, and emoji in my introduction.
I have some experience with RMarkdown already while editing screening questions on html editor for Amazon Mechanical Turk, without knowing that it was RMarkdown features. I think it is very fun to explore what we can do with RMarkdown, and how it could be viewed in HTML website as different formats. It is interesting to see how code interacts with us without us noticing it. During the process, I tried to add colors by using and <span style = "color: red";> to warp text, but it didn't work, but I was able to find a way to insert an emoji into my introduction.
-I am a little bit confused about how many files are needed for homework 1. If I'm understanding it correctly, we only need to edit the readme file right?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,18 @@
...
\ No newline at end of file
+# hw01
+
+## Hello!
+
+My name is **Yirou Fang**. 
+I'm a first year MAPSS student with a concentration in *psychology*.
+This is a really helpful and fun class for me because currently, I'm doing a study in parenting behavior and children's learning through [Amazon Mechanical Turk](https://www.mturk.com), and there will be a lot of necessary data visualization needed for my project.
+
+>Excited! :blush:
```
Comment (deblnia) [inline README.md:9]: Great use of emojis!

**Review by deblnia — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |   X        |              |            |
| **Coding strategy**         |  X         |              |            |
| **Presentation**            |      X     |              |            |
| **Achievement, creativity** |  X         |              |            |
| **Reproducibility**         |     X      |              |            |

Remarks:

* Awesome work! You successfully used five elements of Markdown syntax, including the secret emoji element 👍 💯 🥇 
* I'd just make sure you know how to link images (relative vs. universal file paths) because that can be a little tricky, and images are generally important in any reports you might want to generate using Markdown.

**Comment by choubd** [comment]
Great work! You did an excellent job in using the required elements, adding the emoji and using a sublist. I think there is not much to improve, but I noticed that you mentioned your try using color. Maybe this thread can help https://stackoverflow.com/questions/11509830/how-to-add-color-to-githubs-readme-md-file.

**Comment by tfhernandez** [comment]
Hello Yirou! I really enjoyed reviewing you project and I think you did a great job. In my opinion, the top three strengths of your project were (1) your use of two links, (2) your detailed reflection, and (3) your use of the emoji. If I had to choose one area for improvement, I would suggest adding a few more examples of bold and italicized words. Overall, this was an excellent project!

**Pull request closed without merging**